CONTRIBUTING.md

```diff
@@ -1,13 +1,14 @@
 I've found a bug!
 -----
-Awesome, now let's gather as much information as possible.
+Awesome, but hold up! First of all, make sure that someone else hasn't found this bug previously, if no releated issues found let's gather as much information as possible.
 
-1. Make sure that you using latest version
-```bash
-git pull -u
-rm -r "/Applications/Black Screen"*
-npm run package
-```
+1. Make sure that you are using the latest version.
+
+  ```bash
+  git pull -u
+  rm -r "/Applications/Black Screen"*
```

Comment:
Yes, becouse as for me Black Screen.app lays in /Applications/Black Screen-darwin-x64/Black Screen.app instead of /Applications/Black Screen.app